examples/browser-to-server/tests/test.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-console */
+import { test, expect } from '@playwright/test'
+import { playwright } from 'test-util-ipfs-example'
+import { spawn, exec } from 'child_process'
+import { existsSync } from 'fs'
+
+// Setup
+const play = test.extend({
+  ...playwright.servers()
+})
+
+async function spinUpGoLibp2p() {
+  if (!existsSync('../../examples/go-libp2p-server/go-libp2p-server')) {
+    await new Promise((resolve, reject) => {
+      exec('go build',
+        { cwd: '../../examples/go-libp2p-server' },
+        (error, stdout, stderr) => {
+          if (error) {
+            reject(error)
+            console.error(`exec error: ${error}`)
+            return
```

Comment:
should this throw an error?

Reply:
I just checked the various approached and throwing the error results in better error output.  Thanks @ckousik!